python/hail/matrixtable.py

```diff
@@ -2972,4 +2972,80 @@ def rename(self, fields: Dict[str, str]) -> 'MatrixTable':
 
         return MatrixTable(self._jvds.renameFields(row_map, col_map, entry_map, global_map))
 
+    @typecheck_method(separator=str)
+    def make_table(self, separator='.') -> Table:
+        """Make a table from a matrix table with one field per sample.
```

Comment:
This confused me because I read it as saying that the matrix table has one field per sample.
Also, in the result you have as many fields per sample as the number of entry fields.

How about: "Make a table from a matrix table, with one field for each entry field per sample".